app/pages/property_or_land/PropertyOrLandInternationalAddressPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.property_or_land
+
+import models.InternationalAddress
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+import sections.Assets
+
+final case class PropertyOrLandInternationalAddressPage(index: Int) extends QuestionPage[InternationalAddress] {
+
+  override def path: JsPath = JsPath \ Assets \ index \ toString
+
+  override def toString: String = "propertyOrLandInternationalAddress"
```

Comment:
Rename this to address to standardise with trustees. This will make mapping it to DES easier later on to use existing reads.

Reply:
I should have been explicit, the toString method should be "address" to align with elsewhere in the service.

Reply:
@adamconder, amended. Please review and merge, thanks.